spec/controllers/harvests_controller_spec.rb

```diff
@@ -120,11 +120,11 @@ def valid_attributes
 
       it "re-renders the 'new' template" do
         # Trigger the behavior that occurs when invalid params are submitted
-        Harvest.any_instance.stub(:save).and_return(false)
```

Comment:
we don't need to stub this. the attribute on the next line will fail on their own.